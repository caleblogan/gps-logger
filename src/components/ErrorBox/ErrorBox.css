.ErrorBox {
    position: absolute;
    z-index: 1500;
    top: 0;
    left: 0;
    width: 100%;
}