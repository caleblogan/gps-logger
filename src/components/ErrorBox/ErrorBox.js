import React from 'react';
import PropTypes from 'prop-types';

import styles from './ErrorBox.css';

const ErrorBox = ({children}) => {
  return (
    <div className={styles.ErrorBox}>
      {children}
    </div>
  );
};

ErrorBox.propTypes = {

};

export default ErrorBox;
