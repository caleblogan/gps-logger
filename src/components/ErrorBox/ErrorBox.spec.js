import React from 'react';
import renderer from 'react-test-renderer';
import { shallow, mount, render } from 'enzyme';

import ErrorBox from './ErrorBox';

describe('ErrorBox', () => {
  it('basic snapshot', () => {
    const tree = renderer
      .create(<ErrorBox/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('has correct classNames', () => {
    const tree = renderer
      .create(<ErrorBox/>)
      .toJSON();
    expect(tree.props.className).toBe('ErrorBox')
  })
  it('renders children', () => {
    const errobox = shallow(<ErrorBox><h2>Hey</h2></ErrorBox>);
    expect(errobox.children().length).toBe(1)
  })
  it('renders no children', () => {
    const errobox = shallow(<ErrorBox/>);
    expect(errobox.children().length).toBe(0)
  })
})
