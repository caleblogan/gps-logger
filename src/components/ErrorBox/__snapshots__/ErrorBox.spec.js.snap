// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBox basic snapshot 1`] = `
<div
  className="ErrorBox"
/>
`;
